src/WebJobs.Script.Grpc/Abstractions/IWorkerProvider.cs

```diff
@@ -24,7 +24,7 @@ public interface IWorkerProvider
         /// <param name="config">The host-level IConfiguration.</param>
         /// <param name="logger">The startup ILogger.</param>
         /// <returns>A bool that indicates if the args were configured successfully.</returns>
-        bool TryConfigureArguments(ArgumentsDescription args, IConfiguration config, ILogger logger);
+        bool TryConfigureArguments(WorkerProcessArgumentsDescription args, IConfiguration config, ILogger logger);
```

Comment:
The suffix "Description" could be removed - "WorkerProcessArguments" is more concise

Reply:
done!